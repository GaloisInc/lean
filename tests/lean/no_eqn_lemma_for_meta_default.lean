meta structure foo :=
(x := 10)

#print prefix foo.x._default
