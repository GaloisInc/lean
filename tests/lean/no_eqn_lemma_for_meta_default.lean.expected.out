foo.x._default : â„•
